docs/the_black_code_style/future_style.md

```diff
@@ -52,25 +52,37 @@ tracked in [this issue](https://github.com/psf/black/issues/2188).
 
 ### Removing trailing newlines after code block open
 
-_Black_ will remove trailing newlines after code block openings. That means that the
-following code:
+_Black_ will remove trailing newlines after code block openings. For example:
 
 ```python
 def my_func():
 
     print("The line above me will be deleted!")
-
-    print("But the line above me won't!")
```

Comment:
I think this still has value since it's not immediately clear what "code block opening" _means_.

Reply:
Good observation. I'll change the wording instead, so that it's more explicit